src/Tizen.Multimedia/Common/Display.cs

```diff
@@ -161,8 +163,7 @@ public Display(NUI.Window window, bool uiSync)
             {
                 throw new ArgumentNullException(nameof(window));
             }
-
-            _setter = new EcoreDisplaySetter(window.GetNativeWindowHandler());
+            _setter = new EcoreDisplaySetter(window.GetNativeWindowHandler(), window.WindowPositionSize);
```

Comment:
I recommend you to check that `WindowPositionSize` property returns valid value for current window instance.